openmmtools/tests/test_alchemy.py

```diff
@@ -25,11 +27,15 @@
 
 import nose
 import scipy
+import numpy as np
 from nose.plugins.attrib import attr
 
-from openmmtools import testsystems, forces
-from openmmtools.constants import kB
-from openmmtools.alchemy import *
```

Comment:
ooooof yes lets try and not use `*` import except for in __init__.py to pull everything from a module into a different namesapce.